core/src/components/radio-button-group/radio-button-group.stories.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  title: "Components/Radio Button Group (WIP)",
+};
+
+// TODO: Should probably figure out interactive-friendly way so that one can choose/toggle between radio-buttons?
+
+export const Default = () => `
+  <fieldset class="fudis-radio-button-group fudis-fieldset fudis-input-size__lg">
+    <legend class="fudis-fieldset__legend">
+      <div class="fudis-fieldset__legend__main fudis-fieldset__legend__sm">
+        <div class="fudis-fieldset__legend__main__text-content">
+          <span class="fudis-fieldset__legend__main__text">
+            Legend
+          </span>
+        </div>
+      </div>
+    </legend>
+    <div class="fudis-fieldset-content">
+      <label class="fudis-radio-button">
```

Comment:
Let's add the for="id" attribute pointing to input (All radio-buttons). Also for the first input fields, let's add aria-describedby and link it to guidance id.

Reply:
Added `aria-describedby` to _Guidance_ and _Invalid_ stories.